### PR TITLE
Add pdfium

### PR DESCRIPTION
**Pull request by dsogari**
Add Google's pdfium library from git repo (https://pdfium.googlesource.com/pdfium/)

### CODE REVIEW

**Comment by Alexpux** [comment]
Are there are no Release source?

**Comment by fracting** [comment]
Hi @dsogari, thanks for contribution, there was something wrong in our drone CI server due to my mistake, I've just fixed CI server configuration in https://github.com/Alexpux/MSYS2-packages/pull/501 and https://github.com/Alexpux/MINGW-packages/pull/1199

Could you rebase all your recent PR and force push again to trigger new CI builds?

BTW, I think you need to add `git` as makedepends into your PKGBUILD if it clone source code use git.

**Comment by elieux** [comment]
@fracting, unfortunately Drone seems to fail due to fork errors:
```
5 [main] python2 413 child_info_fork::abort: unable to remap strop.dll to same address as parent (0x260000) - try running rebaseall
```

**Comment by dsogari** [comment]
Thank you @fracting. I finally managed to perform the rebase, but I needed to add the binary attribute to .bz2 files in .gitattributes. I hope this doesn't cause any issues on your side. I think I forgot to add the --force option in the push command. The thing is, I've become frustrated with this overly complex versioning system. It is great, but is giving me headaches.

@Alexpux what do you mean by release build? It thought it was already. Should I add some option to PKGBUILD?

Also, let me express my thankfulness to all of you for this awesome pacman-based repository!

**Comment by fracting** [comment]
@elieux I rebuild the msys32 docker image, this time I run `autorebase.bat` after finishing upgrading msys2 environment, and the fork error gones.

Thanks @jactry for the workaround!

Now CI build passed: https://tea-ci.org/Alexpux/MINGW-packages/376

**Comment by dsogari** [comment]
@Alexpux I understand now what you mean. In their repo they have branches named chromium/<version>, but no tags. I suspect these branches are stable in some sort of way. I could alter the PKGBUILD script to clone the latest branch (chromium/2680, as of this writing) instead of the latest commit. What do you think?

File: mingw-w64-pdfium-git/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Maintainer: Diego Sogari <diego.sogari@gmail.com>
+
+_realname=pdfium
+pkgbase=mingw-w64-${_realname}-git
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
+_basever=2681
```
Comment (Alexpux) [inline mingw-w64-pdfium-git/PKGBUILD]: Don't understand what is the version. Where you get it?

Comment (dsogari) [inline mingw-w64-pdfium-git/PKGBUILD]: Actually, I don't either. It's just a number that appears after the forward slash ('/') in the name of their branches. I took it as a version number.

**Pull request merged**